package.json

```diff
@@ -2388,6 +2387,7 @@
     },
     "optionalDependencies": {
         "canvas": "^2.7.0",
-        "playwright-chromium": "^1.10.0"
+        "playwright-chromium": "^1.10.0",
+        "zeromq": "^6.0.0-beta.6"
```

Comment:
This should go away when I rebase on the other PR I have out. This is so I can run npm install on my machine